.github/workflows/testarch_device.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+export INTERCHANGE_SCHEMA_PATH="$GITHUB_WORKSPACE/env/fpga-interchange-schema/interchange"
+
+curl -O https://capnproto.org/capnproto-c++-0.8.0.tar.gz
+tar zxf capnproto-c++-0.8.0.tar.gz
+cd capnproto-c++-0.8.0
+./configure
+make -j6 check
+sudo make install
+cd ..
+git clone https://github.com/capnproto/capnproto-java.git
+cd capnproto-java
+make -j6
+sudo make install
+cd ..
```

Comment:
Looking at this again, I believe this step is not required. 

capnproto-java is already being installed here and should not actually be needed for the testarch generation https://github.com/SymbiFlow/python-fpga-interchange/blob/875194a9c801429f17a5f2af55e934f706295e0f/.github/workflows/ci.yml#L36

and pycapnp comes from the requirements.txt and might be enough to run the test (this needs to be double-checked though).

In case capnproto-c++ is not required, the `ccache` GH action addition would not be required

Reply:
You are right. I did not export CAPNP_PATH, so I was getting errors, and installing capnproto this way solved it. But with this variable exported this steps are indeed redundant